scripts/setup/install_viewer.sh

```diff
@@ -2,13 +2,19 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 
-set -eux
+set -eu
 
 # Install cbmc-viewer
-if [[ $# -eq 1 ]] ; then
-wget https://github.com/awslabs/aws-viewer-for-cbmc/releases/download/viewer-$1/cbmc_viewer-$1-py3-none-any.whl \
-  && sudo python3 -m pip install --upgrade cbmc_viewer-$1-py3-none-any.whl
-else
-  echo "Error: Specify the version to install"
+
+if [[ $# -ne 1 ]]; then
+  echo "$0: Error: Specify the version to install"
   exit 1
 fi
+
+FILE="cbmc_viewer-$1-py3-none-any.whl"
+URL="https://github.com/awslabs/aws-viewer-for-cbmc/releases/download/viewer-$1/$FILE"
+
+set -x
+
+curl --fail --silent --location "$URL" -o "$FILE"
```

Comment:
Whoops. I was trying to expand my usual `-fsSL` since expanded style seemed preferred here, and I forgot `--show-error`.

Which is your preference, I'm happy any which way, just trying to get a sense of things:

1. Add `--show-error`
2. Go with `-fsSL`
3. Swap back to `wget` and just use `-O`

Reply:
I would go back to using `wget` then.